pkg/vault/vault.go

```diff
@@ -16,8 +23,23 @@ type vaultImpl struct {
 }
 
 func New() VaultClient {
+	env_path := filepath.Join(projectpath.Root, ".env")
```

Comment:
This code is duplicated between here and `create.go`. Consider moving it to its own function.